test/xref2/lib/common.cppo.ml

```diff
@@ -623,7 +627,10 @@ let mkresolver () =
   Odoc_odoc.Resolver.create
     ~important_digests:false
     ~directories:(List.map Odoc_odoc.Fs.Directory.of_string
-#if OCAML_VERSION >= (4,8,0)
+#if OCAML_VERSION >= (5,2,0)
+  (let paths = Load_path.get_paths () in
+   List.filter (fun s -> s <> "") (paths.visible @ paths.hidden))
```

Comment:
This is the conservative version, it is possible that keeping just `paths.visible` would be better for odoc.

Reply:
Apparently, those function are only used in an old `.md` test, so anything is fine.